public/js/locale.js

```diff
@@ -4,6 +4,9 @@
 var lang = 'en'
 var userLang = navigator.language || navigator.userLanguage
 var userLangCode = userLang.split('-')[0]
+if (userLangCode === 'zh') {
+  userLangCode = userLang
+}
```

Comment:
I'm not sure why you add this code here. Can you please explain this change?

Reply:
Sorry, I have re-read too, maybe this code is redundant because had https://github.com/Rwing/hackmd/blob/a03a6d97ffcb02602db35cb084b0bd365479f807/public/js/locale.js#L17